java-checks/src/main/java/org/sonar/java/checks/unused/UnusedPrivateFieldCheck.java

```diff
@@ -180,4 +191,59 @@ private void addAssignment(IdentifierTree identifier) {
     }
   }
 
+
+  private static JavaQuickFix computeQuickFix(VariableTree tree) {
+    Optional<VariableTree> followingVariable = getFollowingVariable(tree);
+    AnalyzerMessage.TextSpan textSpan;
+    if (followingVariable.isPresent()) {
+      textSpan = AnalyzerMessage.textSpanBetween(tree.simpleName(), true, followingVariable.get().simpleName(), false);
+    } else {
+      Optional<SyntaxToken> precedingComma = getPrecedingComma(tree);
+      if (precedingComma.isPresent()) {
+        SyntaxToken endingSemiColon = tree.lastToken();
+        textSpan = AnalyzerMessage.textSpanBetween(precedingComma.get(), true, endingSemiColon, false);
+      } else {
+        textSpan = AnalyzerMessage.textSpanFor(tree);
```

Comment:
I'm wondering if we should not do the same as in "UnusedPrivateMethodCheck": also remove the remaining empty line, by using: `textSpanBetween(QuickFixHelper.previousToken(tree), false, tree, true)`.